LoginPage.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+
+function LoginPage() {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      const response = await axios.post('/api/login', { email, password });
+      if (response.data.success) {
```

Comment:
Avoid the use of window.location.href for navigation. It causes a full page refresh which is not ideal for a single page application. Instead, use the history object from react-router-dom for programmatically changing the route.